x-pack/plugins/secops/public/components/page/network/kpi_network/index.tsx

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import numeral from '@elastic/numeral';
+import React from 'react';
+import { pure } from 'recompose';
+import { KpiNetworkData } from '../../../../graphql/types';
+import * as i18n from './translations';
+
+import {
+  // @ts-ignore
+  EuiCard,
+  EuiFlexGroup,
+  EuiFlexItem,
+} from '@elastic/eui';
+import { has } from 'lodash/fp';
+import { getEmptyTagValue } from '../../../empty_value';
+
+interface KpiNetworkProps {
+  data: KpiNetworkData;
+}
+
+const kpiNetworkCards = (data: KpiNetworkData) => [
+  {
+    title: has('networkEvents', data)
```

Comment:
@FrankHassanabad will be proud, since now we have the value who can be null from the graphql, you should check for the value null `has('networkEvents', data) && data.networkEvents !== null`

Reply:
Thanks @XavierM, nice catch! I've updated accordingly 👍